aeflex/aeflex.go

```diff
@@ -49,9 +53,19 @@ type AEFlexSource struct {
 }
 
 // NewAEFlexSource returns a new discovery object with an authenticated AppEngine client.
-func NewAEFlexSource(project string) (*AEFlexSource, error) {
+func NewAEFlexSource(project, filename string) *AEFlexSource {
 	d := &AEFlexSource{
-		project: project,
+		project:  project,
+		filename: filename,
+	}
+	return d
+}
+
+// Client creates a new initialized AEFlexSource.
+func (client *AEFlexSource) Client() (targetsource.TargetSource, error) {
+	d := &AEFlexSource{
```

Comment:
Add comment explaining why you need a new AEFlexSource instead of just using client?

Also, could this just be copied using copy or assignment? d := *client ?